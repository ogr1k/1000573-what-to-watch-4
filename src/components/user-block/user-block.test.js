import React from "react";
import UserBlock from "./user-block.jsx";
import renderer from "react-test-renderer";


it(`Render Copyright`, () => {
  const tree = renderer
    .create(<UserBlock authorisationStatus="AUTH"/>)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
