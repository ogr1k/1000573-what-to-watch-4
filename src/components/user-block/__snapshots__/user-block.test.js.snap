// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render User block 1`] = `
<div
  className="user-block"
>
  <a
    href="/mylist"
    onClick={[Function]}
  >
    <div
      className="user-block__avatar"
    >
      <img
        alt="User avatar"
        height="63"
        src="/img/avatar.jpg"
        width="63"
      />
    </div>
  </a>
</div>
`;
